analytics/analytics.go

```diff
@@ -289,23 +272,20 @@ func (dc *dataCollection) reportHealthAlert(failurePoint string) {
 }
 
 func (dc *dataCollection) doReportHealthAlert(failurePoint string) error {
-	conn, cancel, err := dc.getGrpcConn()
-	if err != nil {
-		return err
-	}
-	defer cancel()
-	defer conn.Close()
-
 	n := new(pb.NodeHealth)
 	n.BtfsVersion = dc.BTFSVersion
 	n.FailurePoint = failurePoint
 	n.NodeId = dc.NodeID
 	now := time.Now().UTC()
 	n.TimeCreated = now
 
-	ctx, cancel := context.WithTimeout(context.Background(), callTimeout)
-	defer cancel()
-	client := pb.NewStatusClient(conn)
-	_, err = client.CollectHealth(ctx, n)
-	return err
+	config, err := dc.node.Repo.Config()
+	if err != nil {
+		return err
+	}
+	cb := cgrpc.StatusClient(config.Services.StatusServerDomain)
+	return cb.WithContext(context.Background(), func(ctx context.Context, client pb.StatusServiceClient) error {
```

Comment:
it's better to pass a context rather then context.Background(),
since we already have a context.